src/Api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
```

Comment:
Consider using axios instance here, you can read more about it in the notes - https://notes.northcoders.com/courses/js-front-end/axios